Beszel: restarting service after update

C0pywriting opened the pull request:
## ✍️ Description  
Added systemctl command to start beszel.service. This ensures the new version is applied immediately; otherwise, the old version remains in operation until the next restart

C0pywriting committed:
restarting beszel.service after update

Added systemctl command to start beszel.service. This ensures the new version is applied immediately; otherwise, the old version remains in operation until the next restart

MickLesk commented:
maybe this should be an correct rebuild.

msg_info "Stopping Service"
...

...


msg_ok "Started Service" 
like other scripts.

Do you know what i mean? Or any @community-scripts/contributor  can check it

C0pywriting committed:
Add status messages for stopping and starting

Added informational message when stopping the $APP service and a success message when the service starts successfully.

MickLesk commented:
I mean Like this :smile:

https://github.com/community-scripts/ProxmoxVE/blob/main/ct%2Fpaperless-ai.sh

Stopping
Stopped

Update App
Updated App

Starting
Started

C0pywriting committed:
chnage beszel.service

C0pywriting commented:
@MickLesk this is what the current output looks like without my changes updates are made but not applied 


```
____                       __
/ __ )___  _________  ___  / /
/ __  / _ \/ ___/_  / / _ \/ / 
/ /_/ /  __(__  ) / /_/  __/ /  
/_____/\___/____/ /___/\___/_/   


beszel 0.10.1
Checking for updates...
Latest version: 0.10.1
You are up to date
✖️  Currently we don't provide an update function for this Beszel.
 
```

basically the current error message is wrong so i have removed it 

```
    ____                       __
   / __ )___  _________  ___  / /
  / __  / _ \/ ___/_  / / _ \/ / 
 / /_/ /  __(__  ) / /_/  __/ /  
/_____/\___/____/ /___/\___/_/   
                                 
beszel 0.10.1
Checking for updates...
Latest version: 0.10.1
You are up to date
  ✔️  Successfully started the Beszel service.
```

bvdberg01 commented:
I think @MickLesk wants it like this

```
msg_info "Stopping $APP"
systemctl stop beszel-hub.service
msg_ok "Stopped $APP"

msg_info "Updating $APP"
/opt/beszel/beszel update
msg_ok "Updated $APP"

msg_info "Starting $APP"
systemctl start beszel-hub.service
msg_ok "Started $APP"
msg_ok "Update Successful"
```

C0pywriting committed:
Update msg infos

MickLesk committed:
Update beszel.sh

CrazyWolf13 commented on ct/beszel.sh:
```diff
@@ -27,8 +27,18 @@ function update_script() {
+    msg_ok "Updated $APP"
+    
+    msg_info "Starting $APP"
+    systemctl start beszel-hub.service
```
use here without .service too, as you used above without .service

MickLesk committed:
Update beszel.sh

CrazyWolf13 commented on ct/beszel.sh:
```diff
@@ -27,8 +27,18 @@ function update_script() {
+    
+    msg_info "Starting $APP"
+    systemctl start beszel-hub
+    msg_ok "Successfully started $APP"
+    msg_ok "Update Successful"
```
twice msg_ok?
